content/en/blog/_posts/2018-07-16-kubernetes-1.11-release-interview.md

```diff
@@ -0,0 +1,275 @@
+---
+layout: blog
+title: "How the sausage is made: the Kubernetes 1.11 release interview, from the Kubernetes Podcast"
+date:  Monday, July 16, 2018
```

Comment:
Can you change date format to: 2018-07-16

Reply:
Done